packages/client/screens/MusicPlaylistEditorRoomScreen/index.tsx

```diff
@@ -30,6 +30,19 @@ interface AddTrackButtonProps {
     onPress: () => void;
 }
 
+//Should we be using date-fns ?
+//copy pasted from https://stackoverflow.com/questions/19700283/how-to-convert-time-in-milliseconds-to-hours-min-sec-format-in-javascript
+export function msToTime(ms: number): string {
+    const seconds = ms / 1000;
+    const minutes = ms / (1000 * 60);
+    const hours = ms / (1000 * 60 * 60);
+    const days = ms / (1000 * 60 * 60 * 24);
+    if (seconds < 60) return seconds.toFixed(1) + ' Sec';
+    else if (minutes < 60) return minutes.toFixed(1) + ' Min';
+    else if (hours < 24) return hours.toFixed(1) + ' Hrs';
+    else return days.toFixed(1) + ' Days';
+}
```

Comment:
Nice! Dealing with dates is such a nightmare 🥲

With `date-fns` I think it could be written like this:

```suggestion
import { formatDuration, intervalToDuration } from 'date-fns';
import { enUS } from 'date-fns/locale';

export function msToTime(ms: number): string {
    const duration = intervalToDuration({
        start: 0,
        end: ms,
    });
    return formatDuration(duration, { locale: enUS });
}
```